File: 프로그래머스/lv1/12926. 시저 암호/시저 암호.py
def solution(string, n):
    answer = ''
    # print(ord('a'), ord('z'), ord('A'), ord('Z'))
    
    for s in string:
        s = ord(s)
        if 97 <= s <= 122:      # 소문자일 때
            s  = 97 + (s + n - 97) % 26
        elif 65 <= s <= 90:     # 대문자일 때 
            s  = 65 + (s + n - 65) % 26
        # if 97 <= s <= 122 or 65 <= s <= 90:
        #     print(s, n % 25)
        #     s = 97 + (s + n - 97) % 26
        #     print(s)
        answer += chr(s)
    return answer

File: 프로그래머스/1/12926. 시저 암호/시저 암호.java
class Solution {
    public String solution(String s, int n) {
        String answer = "";
        
        for (char c : s.toCharArray()) {
            if (c == ' ') {
                answer += c;
            } else if ('A' <= c && c <= 'Z') {
                answer += (char)('A' + (c + n - 'A') % 26);
            } else {
                answer += (char)('a' + (c + n - 'a') % 26);
            }
            
        }
        
        return answer;
    }
}